messybrainz/db/recording.py

```diff
@@ -148,3 +159,74 @@ def create_recording_clusters():
                 clusters_modified += 1
 
     return clusters_modified, clusters_add_to_redirect
+
+
+def cluster_new_recording(recording_data):
+    """ Tries to associate and cluster newly inserted recording.
```

Comment:
mention that this sends recordings to rabbitmq and is consumed by the clustering script.